### ISSUE TITLE
Doesn't report unexpected Tab character(s) in KDoc

### ISSUE THREAD

**Issue by schalkms** [pinterest/ktlint#569]
Please take a look at the following sample KDoc comments.
None of this lines with tabs gets reported by ktlint.

```Kotlin
package ktlint.sample

/**
 * Tab	 in the middle of this comment.
 * 	Tab at the beginning of this comment.
	* Tab at the beginning of this line.
 * Tab at the end of this line.	
 */
class Test {
}
```

#### Related commit
https://github.com/arturbosch/detekt/commit/9070b81e99af19463e0bb6f963c0519008460613

#### Used version
ktlint 0.34.2

**Comment by paul-dingemans** [comment]
This problem does not occur anymore on current version, as the test below succeeds:
```
    @Test
    fun `format KDoc with tabs`() {
        val code =
            """
            /**
             * Tab${'\t'}in the middle of this comment.
             * ${'\t'}Tab at the beginning of this comment.
            ${'\t'}* Tab at the beginning of this line.
             * Tab at the end of this line.${'\t'}
             */
            class Test {
            }
            """.trimIndent()
        assertThat(IndentationRule().lint(code)).isEqualTo(
            listOf(
                LintError(4, 1, "indent", "Unexpected tab character(s)"),
                LintError(4, 1, "indent", "Unexpected indentation (4) (should be 1)")
            )
        )
    }
```

**Comment by schalkms** [comment]
Alright. Thanks for the update. This is fixed. Therefore, I'm closing this.
